scripts/run_cpp_tests.py

```diff
@@ -91,4 +91,4 @@
   # Save to cache
   pickle.dump(cache, open(args.cache_file, "wb" ))
 
-  exit(exit_code)
+  return exit_code
```

Comment:
Is `return` the correct way to exit Python with an exit code? I thought it was [sys.exit](https://stackoverflow.com/questions/285289/exit-codes-in-python).